packages/bits/demo/src/components/demo/wizard-v2/wizard-dialog/wizard-dialog.example.component.less

```diff
@@ -0,0 +1,57 @@
+@import (reference) "nui-framework-colors";
+@import (reference) "nui-framework-spaces";
+@import (reference) "nui-framework-typography";
+
+
+.step-content-wrapper {
+    padding: @nui-space-md;
+}
+
+.overlay {
+    &-class {
+        display: flex;
+        height: 100%;
+        .setCssVariable(background-color, nui-color-overlay-busy);
+    }
+
+    &-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        max-width: 460px;
+        margin: auto;
+        padding: (@nui-space-md * 2);
+        .setCssVariable(background-color, nui-color-bg-content);
+
+        .header {
+            display: flex;
+            justify-content: center;
+            margin-bottom: @nui-space-md;
+            .nui-text-widget();
+        }
+
+        .body {
+            display: flex;
+            text-align: center;
+            .nui-text-size(default);
+        }
+
+        .footer {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+
+            button[displayStyle="action"] {
+                margin-right: @nui-space-md;
+            }
+        }
+    }
+}
+
+button[displayStyle="action"] {
+    margin-right: @nui-space-md;
+}
+
+h1 {
+    margin-bottom: @nui-space-md;
+}
```

Comment:
Can we remove styles for `button[displayStyle="action"]` (the same in line 44), `h1` (this element doesn`t exist in file.html)?